external/quarkus_quickstarts/README.md

```diff
@@ -0,0 +1,69 @@
+# External (Third Party Container) Tests
+
+Third Party container tests help verify that the AdoptOpenJDK binaries are *good* by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries.  For each application, we choose to run a selection of their functional tests.  
```

Comment:
Start sentence with 
Quarkus quickstart tests are part of the third-party container tests that help verify that the Adoptium binaries ...


and later in paragraph changing AdoptOpenJDK binaries to Adoptium binaries